apps/barnepensjon-ui/src/components/application/Dialogue.tsx

```diff
@@ -32,13 +32,26 @@ export default function Dialogue({ steps, pathPrefix }: DialogueProps) {
         navigate(`${pathPrefix}${step.path}`)
     }
 
+    const [width, setWindowWidth] = useState(0)
+
     const next = currentIndex < steps.length - 1 ? () => visitNavigate(steps[currentIndex + 1]) : undefined
 
     const prev = currentIndex > 0 ? () => visitNavigate(steps[currentIndex - 1]) : undefined
 
+    useEffect(() => {
+        updateDimensions()
+        window.addEventListener('resize', updateDimensions)
+        return () => window.removeEventListener('resize', updateDimensions)
+    }, [])
+
+    const updateDimensions = () => {
+        const width = window.innerWidth
+        setWindowWidth(width)
+    }
+
     return (
         <>
-            <StepIndicator activeStep={currentIndex}>
+            <StepIndicator activeStep={currentIndex} responsive={width < 1024}>
```

Comment:
Magic number

Reply:
Det stemmer. Tok en verdi som var litt større enn stepperen når den er på sitt bredeste. Hvis du har noen andre forslag, så er jeg veldig åpen for det 😄